### PR TITLE
feat: Recon Landing Page

### PR DESCRIPTION
**Pull request by susrithasabbini**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

Recon landing page development

## How did you test it?

Screenshot 2024-12-24 at 3 04 34 PM

## Where to test it?

- [x] INTEG
- [x] SANDBOX
- [x] PROD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/reconScreens/ReconOnboarding/ReconOnbdLanding.res [hunk 1]
```diff
@@ -1,4 +1,27 @@
 @react.component
 let make = () => {
-  <div> {"OnBoarding"->React.string} </div>
+  open PageUtils
+
+  let handleGetStarted = (): unit => {
```
Comment (sagarnaikjuspay) [inline src/reconScreens/ReconOnboarding/ReconOnbdLanding.res:5]: can u remove this log

File: src/reconScreens/ReconOnboarding/ReconOnbdLanding.res [hunk 2]
```diff
@@ -1,4 +1,27 @@
...
+    Js.log("Get Started")
+  }
+
+  <div className="flex flex-col w-full gap-8 items-center justify-center">
+    <PageHeading
+      customHeadingStyle="gap-3 max-w-[860px] flex flex-col items-center"
```
Comment (sagarnaikjuspay) [inline src/reconScreens/ReconOnboarding/ReconOnbdLanding.res:11]: use predefined class names or define this max-w-[860px] value on taiwind cnfig file and use it

File: tailwind.config.js [hunk 3]
```diff
@@ -98,6 +98,7 @@ module.exports = {
         "fs-20": "20px",
         "fs-24": "24px",
         "fs-28": "28px",
+        "fs-48": "48px",
```
Comment (sagarnaikjuspay) [inline tailwind.config.js:101]: max-w-[700px],leading-[60px],max-w-[860px] all this values can be defined here and used

File: src/reconScreens/ReconOnboarding/ReconOnbdLanding.res [hunk 4]
```diff
@@ -1,4 +1,27 @@
...
+      customSubTitleStyle="text-fs-16 text-center max-w-[700px]"
+      subTitle="Effortlessly Track, Match, and Reconcile Transactions with Ease. Gain Real-Time Accuracy and Unmatched Confidence in Financial Operations"
+    />
+    <Button
+      text="Get Started"
+      onClick={_ => handleGetStarted()}
+      buttonType=Primary
+      buttonSize=Large
+      buttonState=Normal
+      customButtonStyle="rounded-sm"
```
Comment (Riddhiagrawal001) [inline src/reconScreens/ReconOnboarding/ReconOnbdLanding.res:23]: Can you check the button rounded

File: src/reconScreens/ReconOnboarding/ReconOnbdLanding.res [hunk 5]
```diff
@@ -1,4 +1,27 @@
...
+    Js.log("Get Started")
+  }
+
+  <div className="flex flex-col w-full gap-8 items-center justify-center">
+    <PageHeading
+      customHeadingStyle="gap-3 max-w-[860px] flex flex-col items-center"
+      title="Effortless Realtime Payment Reconciliation"
+      customTitleStyle="text-fs-48 leading-[60px] text-center max-w-[600px] font-bold"
+      customSubTitleStyle="text-fs-16 text-center max-w-[700px]"
+      subTitle="Effortlessly Track, Match, and Reconcile Transactions with Ease. Gain Real-Time Accuracy and Unmatched Confidence in Financial Operations"
```
Comment (Riddhiagrawal001) [inline src/reconScreens/ReconOnboarding/ReconOnbdLanding.res:15]: Check the font-weight of the subtitle

**Review by JeevaRamu0104 — DISMISSED** [review]

File: src/reconScreens/ReconOnboarding/ReconOnbdLanding.res [hunk 6]
```diff
@@ -1,4 +1,25 @@
...
+    />
+    <Button
+      text="Get Started"
+      onClick={_ => handleGetStarted()}
+      buttonType=Primary
+      buttonSize=Large
+      buttonState=Normal
+      customButtonStyle="rounded-md"
+    />
+    <img alt="sdk" className="sm:w-10/12 -mt-7 w-full" src="/assets/reconLanding.svg" />
```
Comment (Riddhiagrawal001) [inline src/reconScreens/ReconOnboarding/ReconOnbdLanding.res:23]: Please change the alt . Also why do we need this `className="sm:w-10/12 -mt-7`

**Review by JeevaRamu0104 — APPROVED** [review]

**Review by sagarnaikjuspay — APPROVED** [review]

**Pull request merged**